docs/shipping/Code/node-js.md

```diff
@@ -930,6 +930,399 @@ helm uninstall logzio-k8s-telemetry
+
+**collector-config.yaml**
+
+{@include: ../../_include/log-shipping/tracing-shipping/collector-config.md}
+
+This configuration file defines the OpenTelemetry Collector, specifying how to receive, process, and export traces to Logz.io.
+
+**Dockerfile**
+
+```sh
+# Dockerfile for OpenTelemetry Collector
+FROM otel/opentelemetry-collector-contrib:latest
+COPY collector-config.yaml /etc/collector-config.yaml
+CMD ["--config", "/etc/collector-config.yaml"]
+```
+
+#### **4. Build and Push the Docker Image**
+
+To build the Docker image for the Node.js application and OpenTelemetry Collector, use the following commands:
+
+```sh
+cd nodejs-app/
+docker build --platform linux/amd64 -t nodejs-app:latest .
+
+cd otel-collector/
+docker build --platform linux/amd64 -t otel-collector:latest .
+```
+
+Next, push the images to your Amazon ECR repository:
+
+```sh
+# Authenticate Docker to your Amazon ECR repository
+aws ecr get-login-password --region <aws-region> | docker login --username AWS --password-stdin <aws_account_id>.dkr.ecr.<region>.amazonaws.com
+
+# Tag and push the images
+docker tag nodejs-app:latest <aws_account_id>.dkr.ecr.<region>.amazonaws.com/nodejs-app:latest
+docker push <aws_account_id>.dkr.ecr.<region>.amazonaws.com/nodejs-app:latest
+
+docker tag otel-collector:latest <aws_account_id>.dkr.ecr.<region>.amazonaws.com/otel-collector:latest
+docker push <aws_account_id>.dkr.ecr.<region>.amazonaws.com/otel-collector:latest
+```
+
+#### **5. Set Up CloudWatch Log Groups**
+
+- **Log Group Creation**: Create log groups for your Node.js application and OpenTelemetry Collector in CloudWatch.
+
+```sh
+aws logs create-log-group --log-group-name /ecs/nodejs-app
+aws logs create-log-group --log-group-name /ecs/otel-collector
+```
+
+- Ensure the ECS task definition is configured to send logs to the appropriate log groups using the `awslogs` log driver.
+
+#### **6. Create an ECS Cluster and Service**
+
+- **Create ECS Cluster**: Create an ECS cluster using the following command:
+
+```sh
+aws ecs create-cluster --cluster-name app-cluster --region <aws-region>
+```
+
+- **Create ECS Service**: Use the ECS cluster to create a service based on the registered task definition.
+
+```sh
+aws ecs create-service \
+  --cluster <cluster-name> \
+  --service-name <service-name> \
+  --task-definition nodejs-app-task \
+  --desired-count 1 \
+  --launch-type FARGATE \
+  --network-configuration "awsvpcConfiguration={subnets=[\"YOUR_SUBNET_ID\"],securityGroups=[\"YOUR_SECURITY_GROUP_ID\"],assignPublicIp=ENABLED}" \
+  --region <aws-region> \
+```
+
+- **Register Task Definition**: Use the `task-definition.json` file located in the `ecs/` directory to register a new task definition for your Node.js application.
+
+```sh
```

Comment:
I think, try `shell` instead of `sh`

Reply:
You right it looks like `shell` is a more generic identifier that can cover syntax for various shell types